01/assets/js/app.js

```diff
@@ -1,4 +1,41 @@
-const a = '4.2';
-const b = 9;
+const a = "4.2";
+let b = 9;
 
-console.log(a, b);
\ No newline at end of file
+console.log(a, b);
+
+console.log(typeof a, typeof b); // sprawdzenie typu
+
+// Operatory arytmetyczne
+
+const add = parseInt(a) + b;
+const sub = a - b;
+const multi = a * b;
+const split = a / b;
+const perc = b % a;
+let incr = ++b;
+let decr = --b;
+
+const result = {
+  varA: a,
+  varB: b,
+  add: add,
+  sub: sub,
+  multi: multi,
+  split: split,
+  perc: perc,
+  incr: incr,
+  decr: decr,
+};
+
+console.table(result);
+
+const isValueGreaterThanTwenty = (res) =>
+  res > 20 ? ` ${res} is bigger than 20` : `${res} is smaller than 20`;
+
+console.log(isValueGreaterThanTwenty(result.add));
```

Comment:
Można by było użyć pętli `for ... in` :P (innej do obiektó mentee na razie nie znają)